roles/pulp-devel/README.md

```diff
@@ -61,3 +61,4 @@ This role provides the following aliases:
 * `pfixtures`: Run pulp-fixtures container in foreground
 * `pbindings`: Create and install bindings. Example usage: `pbindings pulpcore python`
 * `pminio`: Switch to minio for S3 testing. For stopping it: `pminio stop`
+* `buildgalaxyui`: Build Galaxy UI and move it to static folder
```

Comment:
If I'm reading this right this is plugin-specific yet going into the installer. I think we can't add plugin specific things to the installer even for the devel role. @mikedep333 is this your understanding also? @mikedep333 if so, what are our options to work around?

Reply:
@bmbouter Yeah, I was thinking this too.

However, we already have other plugin-specific aliases:
https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-devel/README.md#aliases

I am OK with another plugin-specific alias, but I want your opinion in light of this.

Reply:
@bmbouter @mikedep333 I removed the alias since we have this now: https://github.com/fabricio-aguiar/galaxy_ng_prerequisites